packages/components/Breadcrumb/stories/index.stories.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import StoryRouter from 'storybook-react-router';
+import Breadcrumb from '../index.js';
+
+export const breadcrumbItems = {
+  projects: [ {
+    name: 'Projects',
+    link: '/projects',
+  }, {
+    name: 'Example project name'
+  } ], paperEdits: [ {
+    name: 'Projects',
+    link: '/projects'
+  },
+  {
+    name: 'Project: The Sample',
+    link: '/projects/the-sample'
+  },
+  {
+    name: 'PaperEdits',
+  },
+  {
+    name: 'Example programme title'
+  }
+  ]
+};
```

Comment:
Have a consistent way of breaking lines, e.g. if at line 12 you are doing `{ [` in a single line, do the same in line 26 and 27 -> `} ];`